state/azure/cosmosdb/cosmosdb.go

```diff
@@ -567,14 +567,27 @@ func (c *StateStore) Multi(ctx context.Context, request *state.TransactionalStat
 	}
 
 	if !batchResponse.Success {
-		// Transaction failed, look for the offending operation
+		var transactionError error
```

Comment:
Is it possible to add a conformance test to ensure all state stores do not error when attempting to delete a non-existent key during a transaction?

https://github.com/dapr/components-contrib/blob/main/tests/conformance/state/state.go

Reply:
Yes, added